test/k8s/service_helpers.go

```diff
@@ -33,12 +33,26 @@ func getTFTPLink(host string, port int32) string {
 		net.JoinHostPort(host, fmt.Sprintf("%d", port)))
 }
 
-func applyPolicy(kubectl *helpers.Kubectl, path string) {
+func applyPolicy(kubectl *helpers.Kubectl, namespace, path string) {
```

Comment:
nit: we probably don't need two methods if this function can take trailing variadic arguments?